src/coordinator/applications.spec.js

```diff
@@ -308,41 +424,64 @@ describe('Coordinator/Applications', function() {
           .stub(objectUtils, 'toCamelCase')
           .returns(expectedFeaturedApplications);
 
-        const applications = new Applications(baseSdk, request, expectedHost);
-        promise = applications.getFeatured(expectedOrganizationId);
-      });
-
-      it('gets the list of featured applications from the server', function() {
-        expect(request.get).to.be.calledWith(
-          `${expectedHost}/organizations/${expectedOrganizationId}/applications/featured`
+        applications = new Applications(
+          baseSdk,
+          request,
+          expectedHost,
+          expectedOrganizationId
         );
       });
 
-      it('formats the list of featured applications', function() {
-        return promise.then(() => {
-          expect(toCamelCase).to.be.calledWith(featuredApplicationsFromServer);
+      context('when the organization ID is provided', function() {
+        beforeEach(function() {
+          promise = applications.getFeatured(expectedOrganizationId);
         });
-      });
 
-      it('returns a fulfilled promise with the featured applications', function() {
-        return expect(promise).to.be.fulfilled.and.to.eventually.deep.equal(
-          expectedFeaturedApplications
-        );
+        it('gets the list of featured applications from the server and does not use the organization ID provided', function() {
+          expect(request.get).to.be.calledWith(
+            `${expectedHost}/applications/featured`
```

Comment:
Shouldn't this end up being `${expectedHost}/v1/{orgId}/applications/featured` for the tenancy urls? I don't see this being set.